code/modules/clothing/suits/modular_armor.dm

```diff
@@ -166,7 +180,7 @@
 	armor = list(
 		melee = ARMOR_MELEE_MINOR,
 		bullet = ARMOR_BALLISTIC_MINOR,
-		laser = ARMOR_LASER_MAJOR,
+		laser = ARMOR_LASER_RIFLE,
```

Comment:
Do you have a reason behind this change? This isn't documented.

Reply:
I honestly think I changed it out of curiosity and forgot to change it back. That was my first endeavor in to coding late at night and was just toying around with things before I got actual help from a server member today.